core/arch/arm/plat-imx/drivers/imx_caam.c

```diff
@@ -63,15 +63,15 @@ static void imx_caam_reset_jr(struct imx_caam_ctrl *ctrl)
 	uint32_t reg_val = 0;
 	uint32_t timeout = 1000;
 
-	io_write32((vaddr_t)&ctrl->jrcfg[0].jrcr, 1);
+	io_write32((vaddr_t)&ctrl->jrcfg[1].jrcr, 1);
```

Comment:
Suggest to use a macro, a global or a argument for identifying the job ring index.